test/unit/edition/has_document_collection_test.rb

```diff
@@ -32,13 +32,6 @@ class Edition::HasDocumentCollectionsTest < ActiveSupport::TestCase
     assert_not edition.reload.part_of_published_collection?
   end
 
-  test "allows assignment of document collection on a saved edition" do
-    edition = create(:imported_publication)
-    document_collection = create(:document_collection, :with_group)
-    edition.document_collection_group_ids = [document_collection.groups.first.id]
-    assert_equal [document_collection], edition.document.document_collections
-  end
-
```

Comment:
Would this test still be relevant/useful if we used an edition in a different state? 🤔 

I'm wondering if we're losing coverage of something useful by removing this test entirely.

What about if we just change `:imported_publication` to `:draft_edition` or some other state?

Reply:
Makes sense @ollietreend - done! 👍